<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpViewer</class>
 <widget class="QDialog" name="HelpViewer">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RoboJournal Documentation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/doc/rj_help.png</normaloff>:/doc/rj_help.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>261</width>
     <height>411</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>571</y>
     <width>800</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777213</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>577</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QToolButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go Back</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/doc/img/prev.png</normaloff>:/doc/img/prev.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Go Forward&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/doc/img/next.png</normaloff>:/doc/img/next.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go to Table of Contents</string>
   </property>
   <property name="text">
    <string>Table of Contents</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>116</width>
      <height>76</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="ChangelogButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;View the list of changes/improvements in this version.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Show Changelog</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/clock-history.png</normaloff>:/icons/clock-history.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HelpViewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HelpViewer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
